File: coding-practice/50 Coding Challenges part 1/C#/problem15.cs
// coding challenge 15 : create a function that will find the nth Fibonacci numbers using recursion
using System;
class problem15
{
    static int fibonacci(int n_fibonacci){
        if(n_fibonacci==0){
            return 0;
        }
        else if(n_fibonacci==1){
            return 1;
        }
        return (fibonacci(n_fibonacci-1) + fibonacci(n_fibonacci-2));
    }

    static void Main(string[] args){
        int n_fibonacci = 10;
        int[] fibonacci_array = new int[n_fibonacci];
        for(int i=0; i<n_fibonacci; i++){
            fibonacci_array[i] = fibonacci(i);
        }
        Console.Write("The first "+n_fibonacci+" numbers are ["+string.Join(", ",fibonacci_array)+"]");
    }
}

File: coding-practice/50 Coding Challenges part 1/JavaScript/scripts/problem15.js
/*Coding challenge #15: Create a function that will find the 
  nth Fibonacci number using recursion. */

function fibonacci(n_fibonacci){
    if(n_fibonacci==0){
        return 0;
    }
    else if(n_fibonacci==1){
        return 1;
    }
    return((fibonacci(n_fibonacci-1) + fibonacci(n_fibonacci-2)));
}
var n = 10;
var number = fibonacci(n-1);
document.write("The"+n+"th Fibonacci number is "+number);